common/genesis/genesis.go

```diff
@@ -45,8 +45,10 @@ func (f *factory) Block(channelID string) *cb.Block {
 	block := protoutil.NewBlock(0, nil)
 	block.Data = &cb.BlockData{Data: [][]byte{protoutil.MarshalOrPanic(envelope)}}
 	block.Header.DataHash = protoutil.BlockDataHash(block.Data)
-	block.Metadata.Metadata[cb.BlockMetadataIndex_LAST_CONFIG] = protoutil.MarshalOrPanic(&cb.Metadata{
```

Comment:
I would say for now and forever.  This is exactly the type of change that will break every existing client.  Great care must be taken in proto modification such as this.  The ramifications are extreme.

Reply:
At some point, we are allowed to break client compatibility -- in v1.2 we removed the old eventhub service in favor of the Deliver API, for instance.  So, I don't think it's out of the realm of possibility that eventually, we stop populating this field.  However, it's a small amount of data, and the benefit to breaking these clients is similarly small.  We really want clients to move to using the other metadata field purely because it's signed and peers validate this signature.

Reply:
Done